test/csuite/random_directio/smoke.sh

```diff
@@ -25,10 +22,10 @@ for threads in $TEST_THREADS; do
 
         # Here are successively tougher schema tests that do not yet
         # reliably pass.  'verbose' can be added to any.
-        #$RUN_TEST -T $threads -S create,create_check       || exit 1
-        #$RUN_TEST -T $threads -S create,drop,drop_check    || exit 1
-        #$RUN_TEST -T $threads -S create,rename             || exit 1
-        #$RUN_TEST -T $threads -S create,rename,drop_check  || exit 1
-        #$RUN_TEST -T $threads -S all,verbose               || exit 1
```

Comment:
If we take the integrated changes, then these can be uncommented, right?

Reply:
If we take the integrated changes, all the new lines with "integrated" are legal, but they won't consistently pass. The old (deleted) lines without "integrated" might pass, except that create_check/drop_check/data_check don't make any sense without "integrated" being set.  I just added a check to disallow that combination.  So the only deleted one that might pass is "create,rename". We don't use rename in MongoDB to my knowledge, so that test seems less important for now.